src/data-dictionary/events/K8sContainerSample/containerOOMEventsDelta.md

```diff
@@ -0,0 +1,9 @@
+---
+name: containerOOMEventsDelta
+type: attribute
+units:
+events:
+  -  K8sContainerSample
+---
+
+Number of "out of memory" events observed for the container (requires k8s v1.24+).
```

Comment:
I think we can remove these types of 'only reported when' notes, just a general recommendation. Reason being: the main reason data dictionary is useful is to let them know what data they see in their system is. if it's not there for whatever reason (like unsupported version) they won't know why it's not there. this helps keeps data clean because sometimes there are many reasons for why an attribute might not be present (e.g., there are a lot of attributes that require specific versions or config settings to be present)

Reply:
@zuluecho9, it seems like if we remove the version information, users definitely won't know why their data isn't appearing, but it sounds like you're saying that there are too many possible reasons why the data doesn't appear, so we should just remove these parenthetical version comments.

Reply:
@zuluecho9 I am going to leave this alone, for now.